readthedocs/config/config.py

```diff
@@ -788,6 +824,60 @@ def validate_python(self):
 
         return python
 
+    def validate_python_install(self, index):
+        """Validates the python.install.{index} key."""
+        python_install = {}
+        key = 'python.install.{}'.format(index)
```

Comment:
Actually, that would be the 3rd element, 0-index p:, yeah I got your point.

Reply:
I was able to change the message to a more friendly way, but! the message that comes from the extra key validation is kind of tricky because we use the dict (here the list is already a dict), we could change this message using a regex (it would work because none of our keys has a number unless someone put a number on purpose). It feels hacky, not sure.

```yaml
python:
  install:
    - path: something
      foo: bar
```

It will be reported as `python.install.0.foo`

Reply:
Oh i see, yeah. I'd say consistency is the most important. If it is going to complicate things in this PR, we can always return to this as a refactor issue. Would you rather just do that?